srv/isc-agent/plugins/tcp/http/models.py

```diff
@@ -0,0 +1,191 @@
+from sqlalchemy import create_engine, MetaData, ForeignKey
+from sqlalchemy import Table, Column, Integer, String, Text, BLOB
+from sqlalchemy.orm import declarative_base, registry
+
+mapper_registry = registry()
+
+Base = mapper_registry.generate_base()
+
+
+# Sigs model
+class Sigs(Base):
+    __tablename__ = 'Sigs'
+
+    id = Column(Integer, primary_key=True)
+    # Should this be Text vs String?
+    patternDescription = Column(Text)
```

Comment:
attribute names should follow a snake case pattern. See https://betterprogramming.pub/string-case-styles-camel-pascal-snake-and-kebab-case-981407998841 for reference